implementation/alpha/node/node.ino

```diff
@@ -287,8 +286,7 @@ void health_packet_generate() {
         do_first_health_packet = false;
 
         Packet new_packet = {
-            // node ID, queue size, 0.0f, 0.0f, 0.0f, 0.0f
-            .reading = {my_id, packet_queue.size, 0.0f, 0.0f, 0.0f, 0.0f},
+            .reading = {(float)packet_queue.size},
```

Comment:
You need to zero out the whole array, otherwise the contents can be whatever is in RAM.  If those random contents turn out to be a large float, print_packet buffer will overflow.